### DISCUSSION TITLE
GeForce Now

### DISCUSSION THREAD

**Discussion by GentMAB** [moraroy/NonSteamLaunchers-On-Steam-Deck#259] [Q&A]
Anyone having an issue installing GeForce Now?  It installs but it never opens anything.

**Comment by moraroy** [comment]
@GentMAB do you have google chrome installed?

**Comment by GentMAB** [comment]
Yes I do

On Sun, Apr 7, 2024 at 12:54 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> do you have google chrome installed?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037527>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV5TIS6GLM7XFJYKQ23Y4GB6BAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXGUZDO>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037527
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
@GentMAB is force compatability unchecked and left blank?

**Comment by GentMAB** [comment]
Yes that is correct as well

On Sun, Apr 7, 2024 at 1:00 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> is force compatability unchecked
> and left blank?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037558>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV4NK5LOG226B332D73Y4GCVXAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXGU2TQ>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037558
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
@GentMAB that's very strange, it may be because Google Chrome isn't installed in the default location. Can you confirm where it's installed at?

**Comment by GentMAB** [comment]
looks like it is running from
/home/deck/.local/share/flatpak/app/com.google.Chrome/current/active/export/share/applications/com.google.Chrome.desktop

On Sun, Apr 7, 2024 at 1:08 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> that's very strange, it may be
> because Google Chrome isn't installed in the default location. Can you
> confirm where it's installed at?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037607>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXVZPEE7DO6QHB3LEOTTY4GDS5AVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXGYYDO>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037607
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
@GentMAB look inside the steam.shortcuts that nsl created and you will be able to see in the Startin and target where it's looking for chrome

**Comment by GentMAB** [comment]
looking for Chrome in /app/bin/chrome

On Sun, Apr 7, 2024 at 1:38 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> look inside the steam.shortcuts
> that nsl created and you will be able to see in the Startin and target
> where it's looking for chrome
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037716>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV26WKYZQWOGHMDUPGTY4GHA3AVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXG4YTM>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037716
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
@GentMAB yeah so that's where chrome should be, does your Google Chrome work I. Desktop mode correctly at all? Or does the nsl shortcuts open chrome in desktop mode at all?

**Comment by GentMAB** [comment]
Doesn't open Chrome in desktop mode at all or gaming mode so it must not be
able to find Chrome

On Sun, Apr 7, 2024 at 1:44 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> yeah so that's where chrome should
> be, does your Google Chrome work I. Desktop mode correctly at all? Or does
> the nsl shortcuts open chrome in desktop mode at all?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037741>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV6VXGR73UT357NKKQ3Y4GHYFAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXG42DC>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037741
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
@GentMAB yeah so your google chrome won't open at all? Even without nsl shortuts, if so you may need to find out why

**Comment by GentMAB** [comment]
Nah Chrome opens fine with out the NSL shortcut or GeForce Now shortcut,
but by itself it opens fine

On Sun, Apr 7, 2024 at 1:49 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> yeah so your google chrome won't
> open at all? Even without nsl shortuts, if so you may need to find out why
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037765>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV2SPT6HGSPRURDAZY3Y4GILZAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXG43DK>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037765
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
@GentMAB weird... nsl only gets that installed location...its the same default location as when steam installs chrome for you on a fresh image, as well as the same location for Google Chrome from the discover store.

**Comment by GentMAB** [comment]
Yeah so the shortcut in GeForce Now shows Target = /usr/bin/flatpak Start
In = /usr/bin but the most up to date version from flathub shows under
/var/lib/flatpak

On Sun, Apr 7, 2024 at 1:52 PM Roy ***@***.***> wrote:

> @GentMAB <https://github.com/GentMAB> weird... nsl only gets that
> installed location...its the same default location as when steam installs
> chrome for you on a fresh image, as well as the same location for Google
> Chrome from the discover store.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037776>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV3HVPBHABB5O5Y2B3LY4GIWRAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXG43TM>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037776
> @github.com>
>


-- 
Steven Green
***@***.***

**Reply by moraroy** [reply]
Try changing the start in and target to reflect your correct location, don't forget to use double quotes for each entry

**Comment by GentMAB** [comment]
Just tried that and got nothing.  Now I think I borked my Discover app b/c
it had a flathub (user) and then Flathub but now the one there I think is
out of date

On Sun, Apr 7, 2024 at 2:04 PM Roy ***@***.***> wrote:

> Try changing the start in and target to reflect your correct location,
> don't forget to use double quotes for each entry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037837>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4SXV7TZHZ5B2BN6KVLFPTY4GKDRAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXHAZTO>
> .
> You are receiving this because you were mentioned.Message ID:
> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037837
> @github.com>
>


-- 
Steven Green
***@***.***

**Comment by GentMAB** [comment]
Everything is installed properly, I think Chrome for user (flathub) has
changed it's location to properly open.  Even trying to add the Chrome
application to Steam itself does not work or open either.

On Sun, Apr 7, 2024 at 2:05 PM Steven Green ***@***.***> wrote:

> Just tried that and got nothing.  Now I think I borked my Discover app b/c
> it had a flathub (user) and then Flathub but now the one there I think is
> out of date
>
> On Sun, Apr 7, 2024 at 2:04 PM Roy ***@***.***> wrote:
>
>> Try changing the start in and target to reflect your correct location,
>> don't forget to use double quotes for each entry
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/discussions/259#discussioncomment-9037837>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AC4SXV7TZHZ5B2BN6KVLFPTY4GKDRAVCNFSM6AAAAABFZ4P6NWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TAMZXHAZTO>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> <moraroy/NonSteamLaunchers-On-Steam-Deck/repo-discussions/259/comments/9037837
>> @github.com>
>>
>
>
> --
> Steven Green
> ***@***.***
>


-- 
Steven Green
***@***.***

**Reply by GentMAB** [reply]
So I was able to launch Chrome by itself before trying to install GeForce Now again and it was showing that it was launching from /usr/bin/flatpak same as the GeForce Now shortcut.  After installing GeForce Now then it doesn't work and Chrome from Steam doesn't work either so something is wrong with how it tries to set it up.

**Reply by moraroy** [reply]
@GentMAB yeah I don't know why, I think it's because like you were saying there is a new directory where chrome installs now
